query/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "databend-query"
-version = "0.1.0"
+version = "22.1.0"
```

Comment:
It is better to have a version specification, and then all crates in the repo should follow

Reply:
This is just a try. In my mind:

> For the first:
> 
> Modify databend version format as year.x.x. Maybe like this 22.0.1
> 
> Or
> 
> year.month.other

Reply:
It looks good to add a version prefix(like '22.') to `get_fuse_version`